SimpleMovies/SimpleMovies/Scenes/List/ListViewControllerCopy.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  ListViewControllerCopy.swift
+//  SimpleMovies
+//
+//  Created by André Sanches Bocato on 21/09/20.
+//  Copyright © 2020 Eduardo Sanches Bocato. All rights reserved.
+//
+
+import UIKit
+
+class ListViewControllerCopy: UIViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var searchTextField: UITextField!
+    @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Properties
+    
+    private var searchResults = [SearchResponse.Result]()
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        FavoritesManager.shared.load()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+    
+    // MARK: - Setup
+    
+    private func setupTableView() {
+        tableView.dataSource = self
+        tableView.isHidden = true
+        tableView.estimatedRowHeight = 200
+        tableView.rowHeight = 200
+        tableView.register(
+            UINib(nibName: "ListTableViewCell", bundle: Bundle(for: ListTableViewCell.self)),
+            forCellReuseIdentifier: "ListTableViewCell"
+        )
```

Comment:
[SUGGESTION]
É preferível usar o método já implementado na nossa API, que simplifica o registro de uma célula.

```suggestion
        tableView.register(cell: ListTableViewCell.self)
```